README.md

```diff
@@ -1 +1,69 @@
-# Leaderboard
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Leaderboard
+
+
+## Built With
+
+- Javascript
+- HTML and CSS
+
+
+[Live Demo Link](Not yet)
+
+
+## Getting Started
+
+**This project is part of a training series to highlight the need for proper documentation, best practices, and writing clean codes with html, css and Javascript.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+- Have a good working knowledge of Git and Github
+### Setup
+- Open a command line terminal (Git bash preferrably) and navigate to a directory where you would like to save the work folder using cd.
+- Clone the repository with git clone https://github.com/wale-prog/To-do-list.git
+-To get the cuurent working tree, git fetch --all, then checkout to "todo-list-structure" branch.
```

Comment:
- [ ] [OPTIONAL] Please, consider fixing the correct repo link - it seams this link is came from the old project "To do list"